### ISSUE TITLE
Backport edf1c6f0baa5d421554db057eff2a0943d928c77

### ISSUE THREAD

**Issue by cwfitzgerald** [BVE-Reborn/rend3#349]
Should be backported to v0.2 to unblock @John-Nagle while #348 is looked at.

**Comment by John-Nagle** [comment]
Yes, no longer needed. That was to get past a bug last February.
